tests/Assetic/Test/Extension/Twig/TwigResourceTest.php

```diff
@@ -24,9 +24,9 @@ protected function setUp()
 
     public function testInvalidTemplateNameGetContent()
     {
-        $loader = $this->getMock('Twig_LoaderInterface');
+        $loader = $this->getMock('Twig_LoaderInterface', array('getSource', 'getSourceContext', 'getCacheKey', 'isFresh', 'exists'));
```

Comment:
Unfortunately for the Travis check with dev dependencies `getSourceContext` is in that loader: https://github.com/twigphp/Twig/blob/2.x/lib/Twig/LoaderInterface.php#L28

I prefered to do this instead of modifying test scenarios (maybe we could drop the support for Twig 2.0 in 1.* and introduce version 2.* supporting only Twig 2.0 - when it's released - but I'm not the one to decide).